go/engine/pgp_encrypt.go

```diff
@@ -163,6 +179,10 @@ func (e *PGPEncrypt) Run(m libkb.MetaContext) error {
 		}
 	}
 
+	for _, warning := range e.warnings.Strings() {
+		m.Warning(warning)
```

Comment:
Don't do this to communicate with a user, this a log message.  The fact that it shows up in the CLI is incidental.

You should store any warnings in the engine type (like `warnings` in the PGPSignEngine, but expose it by making it `Warnings`).  After calling Run on the engine, check for warnings.  Put them in a result type avdl so clients can get them.

Reply:
OK then, I'll add a new method to the PGP UI.